phpBB/phpbb/install/event/kernel_exception_subscriber.php

```diff
@@ -0,0 +1,125 @@
+<?php
+/**
+ *
+ * This file is part of the phpBB Forum Software package.
+ *
+ * @copyright (c) phpBB Limited <https://www.phpbb.com>
+ * @license GNU General Public License, version 2 (GPL-2.0)
+ *
+ * For full copyright and license information, please see
+ * the docs/CREDITS.txt file.
+ *
+ */
+
+namespace phpbb\install\event;
+
+use phpbb\exception\exception_interface;
+use phpbb\install\controller\helper;
+use phpbb\language\language;
+use phpbb\template\template;
+use Symfony\Component\EventDispatcher\EventSubscriberInterface;
+use Symfony\Component\HttpKernel\Exception\HttpExceptionInterface;
+use Symfony\Component\HttpKernel\KernelEvents;
+use Symfony\Component\HttpKernel\Event\GetResponseForExceptionEvent;
+
+/**
+ * Exception handler for the installer
+ */
+class kernel_exception_subscriber implements EventSubscriberInterface
+{
+	/**
+	 * @var helper
+	 */
+	protected $controller_helper;
+
+	/**
+	 * @var language
+	 */
+	protected $language;
+
+	/**
+	 * @var template
+	 */
+	protected $template;
+
+	/**
+	 * Constructor
+	 *
+	 * @param helper	$controller_helper
+	 * @param language	$language
+	 * @param template	$template
+	 */
+	public function __construct(helper $controller_helper, language $language, template $template)
+	{
+		$this->controller_helper	= $controller_helper;
+		$this->language				= $language;
+		$this->template				= $template;
+	}
+
+	/**
+	 * This listener is run when the KernelEvents::EXCEPTION event is triggered
+	 *
+	 * @param GetResponseForExceptionEvent	$event
+	 */
+	public function on_kernel_exception(GetResponseForExceptionEvent $event)
```

Comment:
Similar to the above use of "is", the "on" is IMHO useless and doesn't add any info.